### PR TITLE
Empty all of the GNL

### PR DESCRIPTION
**Pull request by OscarMulder**
- Fixes the emptying of GNL
- Free's the line
- My code is still broken
:octocat:

### CODE REVIEW

**Review by Eathox — APPROVED** [review]

File: tests.c [hunk 1]
```diff
@@ -22,7 +38,7 @@ void bonus_test_one()
 
 	fd1 = open("test_files/easy", O_RDONLY);
 	fd2 = open("test_files/stuff", O_RDONLY);
-
+	// dprintf(2, "Fd's: %d, %d\n", fd1, fd2);
```
Comment (Eathox) [inline tests.c:41]: Why add commented debugs?

File: tests.c [hunk 2]
```diff
@@ -46,7 +62,7 @@ void bonus_test_two()
 	fd1 = open("test_files/easy", O_RDONLY);
 	fd2 = open("test_files/stuff", O_RDONLY);
 	fd3 = open("test_files/data", O_RDONLY);
-
+	//dprintf(2, "Fd's: %d, %d, %d\n", fd1, fd2, fd3);
```
Comment (Eathox) [inline tests.c:65]: Why add commented debugs?

**Review by saarikoski-jules — APPROVED** [review]
Thanks for not closing my fds.

File: tests.c [hunk 3]
```diff
@@ -77,7 +93,8 @@ void bonus_test_three()
 	fd1 = open("test_files/easy", O_RDONLY);
 	fd2 = open("test_files/standard", O_RDONLY);
 	fd3 = open("test_files/data", O_RDONLY);
-
+	// dprintf(2, "Fd's: %d, %d, %d\n", fd1, fd2, fd3);
```
Comment (Eathox) [inline tests.c:96]: Why add commented debugs?

**Review by Eathox — CHANGES_REQUESTED** [review]

**Pull request merged**